src/components/atomic/az-list/AZList.jsx

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useTracking } from 'react-tracking';
+
+import { AZListArray } from '../../../constants';
+import { useAppPaths } from '../../../hooks';
+
+const AZList = ({ azListLabel = 'Browse' }) => {
+	const { ExpandPath } = useAppPaths();
+	const tracking = useTracking();
+
+	useEffect(() => {
+		AZListArray.push('All');
+	}, []);
+
+	const handleAZLinkClick = (letter) => {
+		tracking.trackEvent({
+			type: 'Other',
+			event: 'DrugDictionaryApp:Other:AZClick',
+			linkName: 'DrugDictionarySearchAlphaList',
+			letter,
```

Comment:
The event should have one more param : |analyticsName     | Drug |

Reply:
Added